### PR TITLE
added lights first draft

### PR DESCRIPTION
**Pull request by Alexwen709**
we need the imports, everything else should be fine unless changes are requested

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,139 +1,283 @@
...
+    global robot_state
+    global robot_mode
+    robot_state = get_robot_state()
+    robot_mode = get_robot_mode()
+
+    # Run the lights (actual displaying)
+    try:
+        match robot_mode:
+            case RobotMode.NoCode:
+                # Runs when there is no code on the RoboRIO, STATIC BLACK
```
Comment (aschokking) [inline main.py]: ⭐  Using 'black' which is basically no lights on means it's impossible to tell the difference between this state and the lights just being totally broken and not working?

**Review by aschokking — APPROVED** [review]
Looks like a great start! Have you been able to test that the code works on the lights given different modes/states being set?

File: main.py [hunk 2]
```diff
@@ -1,139 +1,217 @@
-import time
 
+# PIXEL_PIN = board.D5
```
Comment (aschokking) [inline main.py]: ⭐ just delete this commented out code, it's mostly duplicated anyway?

File: main.py [hunk 3]
```diff
@@ -1,139 +1,217 @@
...
+print(sys.version)
+import time
 from digitalio import DigitalInOut, Direction, Pull
 import board
 import neopixel
 from rainbowio import colorwheel
 
+
+# Stuff from 2023, might need to be updated
+# connect the pin to D5
```
Comment (aschokking) [inline main.py]: ⭐  it is d5, so this comment can go now?

**Review by aschokking — APPROVED** [review]

File: main.py (hunk 3 above)
Comment (Alexwen709) [reply main.py]: 👍

File: main.py (hunk 2 above)
Comment (Alexwen709) [reply main.py:2]: 👍

**Pull request merged**